test/integration/users.test.js

```diff
@@ -1040,4 +1037,52 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("POST /users/migrate", function () {
+    it("Should return 401 when user is unauthorize", function (done) {
+      chai
+        .request(app)
+        .post("/users/migrate")
+        .end((err, res) => {
+          if (err) {
+            return done();
+          }
+          expect(res).to.have.status(401);
+          expect(res.body.message).to.equal("Unauthenticated User");
+          return done();
+        });
+    });
+    it("Should update the user", async function () {
+      const response1 = await chai
+        .request(app)
+        .post(`/users/migrate`)
+        .set("Cookie", `${cookieName}=${superUserAuthToken}`);
+      expect(response1).to.have.status(200);
+      expect(response1.body).to.eql({
+        message: `All Users github_user_id added successfully`,
+      });
+      const response2 = await chai.request(app).get(`/users`).set("cookie", `${cookieName}=${superUserAuthToken}`);
```

Comment:
```suggestion
      const usersReponse= await chai.request(app).get(`/users`).set("cookie", `${cookieName}=${superUserAuthToken}`);
```

Reference:
https://github.com/Real-Dev-Squad/website-backend/blob/421c736ef0f9c4250cf28b0900c5ba26e80e4e42/test/integration/users.test.js#L343

Reply:
Sure I'll make the changes